tests/ut_main.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+import database
+import EScavange
+import gamemaker
+
+class TestDatabaseCreation(unittest.TestCase):
+  #Test that we can access the database. Should not test that the database DS is correct, that should be
+  #covered by the database tests.
+  
+  def setUp(self):
+    self.db = database.Database()
+    self.gameMaker = gamemaker.GameMaker()
+  
+  def accessDB(self):
+    pass
+    #stub for now : check that we can access the database
+    # How do i verify that the database is open???
+
+class TestLogin(unittest.TestCase):
+  #Test log in functionality. Will determine if username belongs to gamemaker or team and call correct method
+  
+  def setUp(self):
+    self.db = database.Database()
+    self.inputs = EScavange.main()
+    self.inputs.user = "Bob"            #Set the user inputs (so the test can be automatic, no user input necessary)
+    self.inputs.password = "password"
+        
+  def testLoginNoTeams(self):
+    self.assertEquals(EScavange.login(), "There is no such user. Please create a new team", "Login did not give the correct error message")
+
+  def testLoginTeam(self):
+    self.db.add_team(self.inputs.user,self.inputs.password) #What is the syntax to add a team to the DB?????????
+    self.assertEquals(EScavange.login(), "Currently logged in as Bob", "Login was not with the correct user")
+    
+  def testLoginMasterFAIL(self):
+    self.inputs.user = "maker"
+    self.inputs.password = "wrongpassword"
+    self.assertEquals(EScavange.login(), "Failed to log in as Master", "Was able to log in as master and should not have been able to")
+    
+  def testLoginMaster(self):
+    self.inputs.user = "maker"        #is this what we decided for username and password
+    self.inputs.password = "password"
+    self.assertEquals(EScavange.login(), "Logged in as Master. What would you like to do? [add-team, add-landmark, add-game]", "Was NOT able to log in as master and should have been able to")
+ 
+class TestMain(unittest.TestCase):
+  #Test prompts and functionality of main script
+  
+  def setUp(self):
+    self.db = database.Database()
+    self.inputs = EScavange.main()
+   
+  def testPromptLogin(self):
+    self.assertEquals(EScavange.main(), "Welcome to EScavange! Please log in: ", "Main function does not print expected value")
+    
+  def testPromptAsGameMaker(self):
+    # Should I continue with this? Should I move game maker & team to another class? :( 
+    self.inputs.user = "maker"
+    self.inputs.password = "password"
+    EScavange.login()
+    
+    self.assertEquals(self.inputs.cmd="add-team"), "You chose to add a team. Please enter the username: ", "add-team does not work")
```

Comment:
The cmd string can be separate words (ie. "add team" as opposed to "add-team"). Also, will the cmd function really be returning a string or just outputting it to the console? My guess is outputting to console, and if that's the case, we can't exactly test the message that will be printed here. Same for any of the above things you're testing.